site/en/setup.md

```diff
@@ -166,23 +189,43 @@ made in containers back to your local Docker image.)
 
 ### Testing the Docker installation
 
-To start, download and extract a document from the [OCR-D GT Repo](https://ocr-d-repo.scc.kit.edu/api/v1/metastore/bagit/):
+To start, download and extract a document from the [OCR-D GT Repo](https://ola-hd.ocr-d.de/search?q=&fulltextsearch=false&metadatasearch=false&isGT=true&perPageRecords=30):
 
 ```sh
-wget 'https://ocr-d-repo.scc.kit.edu/api/v1/dataresources/736a2f9a-92c6-4fe3-a457-edfa3eab1fe3/data/wundt_grundriss_1896.ocrd.zip'
-unzip wundt_grundriss_1896.ocrd.zip
+wget "https://ola-hd.ocr-d.de/api/export?id=21.T11998/0000-001C-F82E-8&internalId=false" -O wundt_grundriss_1896.ocrd.zip
+sudo unzip wundt_grundriss_1896.ocrd.zip
 cd data
 ```
 
-Let's segment the images in file group `OCR-D-IMG` from the zip file into regions, thereby creating a 
-[PAGE-XML](https://github.com/PRImA-Research-Lab/PAGE-XML) file group `OCR-D-SEG-BLOCK-DOCKER`)
+Now, spin up the docker container:
 
-You can spin up a Docker container, mounting the current working directory like this:
+```sh
+docker run --user $(id -u) --workdir /data --volume $PWD:/data -it ocrd/all bash
```

Comment:
```suggestion
docker run --user $(id -u) --workdir /data --volume $PWD:/data --rm -it ocrd/all bash
```

Otherwise the container will be retained, cluttering the file system. Or was that the intention here that the container should not be removed after use?

Reply:
That was not intended, I'll commit your suggestion, thanks!